backend/config/curation-api.yml

```diff
@@ -200,16 +199,14 @@ paths:
                   - contact_name
                   - contact_email
                   - publisher_metadata
-        "201":
-          $ref: "#/components/responses/201"
         "400":
           $ref: "#/components/responses/400_multiple"
         "401":
           $ref: "#/components/responses/401"
         "403":
           $ref: "#/components/responses/403"
-        "404":
-          $ref: "#/components/responses/404"
+        "405":
+          $ref: "#/components/responses/405"
```

Comment:
Can you say a bit about why you added this here? The only reason I can come up with is that because _some_ http methods are not allowed for this endpoint, a 405 is a possibility. 

But it would stand to reason that this should therefore be a possible response for all of our endpoints (since we don't have any endpoints which accept every http method).

Reply:
If the collection is not private or a revision.